eemeter/co2/cache.py

```diff
@@ -0,0 +1,110 @@
+import os
+import json
+from sqlalchemy import (
+    create_engine,
+    Table,
+    MetaData,
+    Column,
+    Integer,
+    String,
+)
+from sqlalchemy.sql import select
+import pandas as pd
+
+
+class SqlCO2Store(object):
```

Comment:
This looks pretty similar to the other store, I wonder if it's worth trying to factor out the common code. Doesn't bother me too much to leave it as long as it's tested. I think it's worth adding some unit tests for this as well - if you need ideas check out the tests for the object upon which this one is based.